src/Microsoft.ComponentDetection.Detectors/npm/NpmLockfile3Detector.cs

```diff
@@ -0,0 +1,160 @@
+﻿namespace Microsoft.ComponentDetection.Detectors.Npm;
+
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.ComponentDetection.Contracts;
+using Microsoft.ComponentDetection.Contracts.TypedComponent;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json.Linq;
+
+public class NpmLockfile3Detector : NpmLockfileDetectorBase, IExperimentalDetector
```

Comment:
Do you need to add this to the dependency injection configuration?

Reply:
Yep! Missed that